### PR TITLE
UI implementation for system messages on timed conversations

### PR DESCRIPTION
**Pull request by nicorsm**
## What's new in this PR?

Added code to handle timed conversations from UI side. Work in progress, needs more fixes, testing and cleanup.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -19,11 +19,6 @@ DEPENDENCIES:
   - RBBAnimation (from `https://github.com/wireapp/RBBAnimation`, commit `7dd8d9a3cf610be5f7c1e4459692d555d70704c7`)
   - SCSiriWaveformView (from `https://github.com/wireapp/SCSiriWaveformView`, tag `v1.0.3`)
 
-SPEC REPOS:
-  https://github.com/cocoapods/specs.git:
-    - GoogleAPIClient
-    - GTMSessionFetcher
-
```
Comment (mikeger) [inline Podfile.lock]: Please update your cocoapods!

File: Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme [hunk 2]
```diff
@@ -112,7 +112,7 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "-ZMBackendEnvironmentType staging"
-            isEnabled = "NO">
+            isEnabled = "YES">
```
Comment (mikeger) [inline Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme]: Please reset the prod BE per default.

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/MessageTimerUpdateCell.swift [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        }
+        
+    }
+
+}
+
+
+
+/*
+class MissingMessagesCell: IconSystemCell {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/MessageTimerUpdateCell.swift]: Comment left.

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/MessageTimerUpdateCell.swift [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+import UIKit
+
+class MessageTimerUpdateCell: IconSystemCell {
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/MessageTimerUpdateCell.swift]: Empty methods: most likely can be removed.

**Comment by mikeger** [comment]
@nicorsm ready to go? 🔥 I see the conflict

**Comment by nicorsm** [comment]
@mikeger yep, just need to merge the PR in datamodel

**Pull request merged**